src/main/java/edu/ucsb/cs156/courses/entities/FinalsEntity.java

```diff
@@ -0,0 +1,22 @@
+package edu.ucsb.cs156.courses.entities;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import java.time.LocalDateTime;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity(name = "ucsb_finals")
+public class FinalsEntity {
+  @Id private String quarterYYYYQEnrollCd;
+  private String hasFinals;
+  private String comments;
+  private LocalDateTime finalExamStart;
+  private LocalDateTime finalExamEnd;
+}
```

Comment:
```suggestion
  @LastModifiedDate
  private LocalDateTime lastModified;
}
```

Reply:
Since the issue says this:
 > if the entry in the database table is present, but more than COURSE_DATA_STALE_THRESHOLD_MINUTES old, then the api endpoint will be queried again, the value stored in the database table, and the result returned. (Note that you can get the value of COURSE_DATA_STALE_THRESHOLD_MINUTES by invoking the getCourseDataStaleThresholdMinutes method of the IsStaleService.)

That means that the database entity needs to have a `lastModified` field in it.

You can use the techniques described here to add that to the entity:
https://ucsb-cs156.github.io/topics/spring_boot/spring_boot_database.html#timestamps-on-database-rows

That way the "last updated" field is updated automatically.